### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by stricklerj**
Modification to only show resource management task and pause the simulation. Basic post processing of log data to evaluate user performance over time.

### CODE REVIEW

**Comment by remod** [comment]
Am I assuming correctly that this software can be hacked and does not need to fulfill a certain quality level?

**Comment by remod** [comment]
It might be helpful to add a section at the start of the README listing what you changed with respect to the original version. Bullet points should be sufficient. Then we don't need to reverse engineer when looking at this fork in one year from now :)

File: OpenMATB.py [hunk 1]
```diff
@@ -888,16 +888,16 @@ def getConfigValue(key, defaultvalue):
     app = QtWidgets.QApplication(sys.argv)
 
     loadConfig()
+    #scenario_FullPath, none = QtWidgets.QFileDialog.getOpenFileName(
+    #    None, VERSIONTITLE + ' - ' + _('Select a scenario'), SCENARIOS_PATH, "(*.txt)")
 
-    scenario_FullPath, none = QtWidgets.QFileDialog.getOpenFileName(
-        None, VERSIONTITLE + ' - ' + _('Select a scenario'), SCENARIOS_PATH, "(*.txt)")
-
+    scenario_FullPath = '/home/jon/nasa_tlx/OpenMATB/Scenarios/plant.txt'
```
Comment (remod) [inline OpenMATB.py]: Can we find a better solution that this software can be started on any laptop?

Comment (stricklerj) [reply OpenMATB.py]: yes the file browser is now running

Comment (remod) [reply OpenMATB.py]: nice!

**Review by remod — APPROVED** [review]

**Pull request merged**